### ISSUE TITLE
Pylon positions are shifted in hand/inventory

### ISSUE THREAD

**Issue by novasolarius** [williewillus/Botania#252]
In the inventory and on the hotbar the mana, Gaia and natura pylons are rendered about a quarter slot to the right, causing some slight overlap. In the hand however they are positioned a significant distance to the left, which is best seen when holding one in the off-hand.

I have uploaded a pair of images here: http://imgur.com/a/WrOIr

They render correctly when placed. Also, is it intended that the natura pylon has a ring around it in the inventory but not when placed?

**Comment by howtonotwin** [comment]
Not sure if relevant, but OBJLoader complains that command `s` is not supported during startup.

**Comment by williewillus** [comment]
The natura having a ring in the inv is minor but separate issue. Kinda waiting on forge to add a new feature to let me hide it without using code (just json)

**Comment by novasolarius** [comment]
Fair enough. I'm just glad the positioning issue was fixed. Thanks for that, by the way.
